FreeRTOS/Source/portable/GCC/RISC-V/portASM.S

```diff
@@ -957,7 +957,7 @@ xPortSandboxReturnTrampoline:
 .align 4
 .type pxPortSandboxGetReturnTrampoline, %function
 pxPortSandboxGetReturnTrampoline:
-	clgc              ca0, xPortSandboxReturnTrampoline
+	cllc              ca0, xPortSandboxReturnTrampoline

```

Comment:
We can probably also use `pvAlmightyCodeCap` here instead

Reply:
For what?

Reply:
Just another way to do it

Reply:
I don't understand what you're proposing. `pvAlmightCodeCap` will have address 0, you can't just replace `xPortSandboxReturnTrampoline` with it.

Reply:
Ah, sorry I wasn't clear. I meant to use `pvAlmightCodeCap` for deriving/building the cap here (i.e., its setting the address) the same way I did in the C code, but `cllc` seems to be more efficient to me here.